frontend/src/components/core/Sidebar/SidebarNav.test.tsx

```diff
@@ -90,10 +89,10 @@ describe("SidebarNav", () => {
   it("replaces underscores with spaces in pageName", () => {
     const wrapper = shallow(<SidebarNav {...getProps()} />)
 
-    const links = wrapper.find(StyledSidebarNavLink).find(OverflowTooltip)
+    const links = wrapper.find(StyledSidebarNavLink).find(StyledSidebarNavLink)
```

Comment:
I think this doesn't make sense to find  `StyledSidebarNavLink` in `StyledSidebarNavLink`. Do you mean: `StyledSidebarLinkText`?

Reply:
🤦🏻‍♂️ haha sorry for the poor proofreading. Names were too similar! Updated in [3062906](https://github.com/streamlit/streamlit/pull/5538/commits/30629066943913784317cae7fac9555e12f301e9)